scripts/qa_crowbarsetup.sh

```diff
@@ -895,6 +895,12 @@ function onadmin_set_source_variables
             CLOUDISONAME=${want_cloud8_iso:="SUSE-OPENSTACK-CLOUD-CROWBAR-8-${arch}*1.iso"}
             CLOUDLOCALREPOS="SUSE-OpenStack-Cloud-Crowbar-8-official"
         ;;
+        RC*)
+            cs=${cloudsource/#RC/RC}
```

Comment:
I think it should be merged into the `GMC*|M?` case above.  and I think, the substitution here is even a no-op and can be omitted.